locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -499,9 +499,9 @@ export const notifyNewKeyToWedlocks = async (key: Key, network: number) => {
         // add event details props
         eventName: eventDetail?.eventName,
         eventDate: eventDetail?.eventDate,
-        eventDescription: eventDetail?.eventDescription,
         eventTime: eventDetail?.eventTime,
         eventAddress: eventDetail?.eventAddress,
+        eventUrl: 'eventDetail?.eventUrl',
```

Comment:
This needs to be completed. For this I will need https://github.com/unlock-protocol/unlock/pull/12936 to be merged

Reply:
fixed!